example/objc/TextSort/TXSTextboxListenerImpl.swift

```diff
@@ -0,0 +1,22 @@
+//
```

Comment:
Can you put new hand-written source files under example/handwritten-src?  It's important for clarity that a new developer be able to know what kind of source they're looking at.  (I'd suggest a new directory handwritten-src/swift in this case.)